introduction.md

```diff
@@ -8,9 +8,9 @@ If you have suggestions to improve the guidelines, or questions about them, plea
 
 <h2>Developer Expectations</h2>
 
-Developers, all users with commit access, and all users who officially support a plugin are expected to abide by the Directory Guidelines. 
+Developers, all users with commit access, and all users who officially support a plugin must abide by the Directory Guidelines.
 
-Violations may result in plugins or plugin data (for previously approved plugins) being removed from the directory until the issues are resolved. Plugin data, such as user reviews and code, may not be restored depending on the nature of the violation and the results of a peer-review of the situation. Repeat violations may result in all the author’s plugins being removed and the developer being banned from hosting plugins on WordPress.org.
+Violations of these guidelines can result in plugins or plugin data (for previously approved plugins) being removed from the directory until the issues are resolved. Plugin data, such as user reviews and code, may not be restored depending on the nature of the violation and the results of a peer-review of the situation. Repeat violations may result in all the author’s plugins being removed and the developer being banned from hosting plugins on WordPress.org.
```

Comment:
Recommend parenthesis instead of command around the “such as” statement.

Recommend adding “of” to “in all the author’s plugins”.

Wondering if clarity should be made between a plugin author and a plugin owner? Many users can author a plugin, so who is ultimately held responsible? Just the owner? All authors? Some authors more than others? Etc…

Reply:
Fixed paren, thank you!

As for the clarity, that could be squeezed into #2 (responsibility for plugin actions). The official answer is "anyone who has been added as a committer and/or support person to a plugin" ... That's a much larger issue to tackle and I'm aiming for that in the next revisit. It never ends :)